### PR TITLE
Update for newest libnx

### PR DESCRIPTION
**Pull request by averne**

### CODE REVIEW

File: include/tesla.hpp [hunk 1]
```diff
@@ -3436,7 +3436,7 @@ namespace tsl {
             SharedThreadData *shData = static_cast<SharedThreadData*>(args);
 
             // To prevent focus glitchout, close the overlay immediately when the home button gets pressed
-            hidsysAcquireHomeButtonEventHandle(&shData->homeButtonPressEvent);
+            hidsysAcquireHomeButtonEventHandle(&shData->homeButtonPressEvent, true);
             eventClear(&shData->homeButtonPressEvent);
```
Comment (masagrator) [inline include/tesla.hpp:3440]: This line is not needed because autoclear bool do it automatically

File: include/tesla.hpp [hunk 2]
```diff
@@ -3462,7 +3462,7 @@ namespace tsl {
             SharedThreadData *shData = static_cast<SharedThreadData*>(args);
 
             // To prevent focus glitchout, close the overlay immediately when the power button gets pressed
-            hidsysAcquireSleepButtonEventHandle(&shData->powerButtonPressEvent);
+            hidsysAcquireSleepButtonEventHandle(&shData->powerButtonPressEvent, true);
             eventClear(&shData->powerButtonPressEvent);
```
Comment (masagrator) [inline include/tesla.hpp:3466]: This line is not needed because autoclear bool do it automatically

Comment (averne) [reply include/tesla.hpp:3466]: Changed to false since we're doing manual clearing

**Comment by WerWolv** [comment]
Also handled by #46

**Pull request closed without merging**